instrumentation/sidekiq/lib/opentelemetry/instrumentation/sidekiq/middlewares/server/tracer_middleware.rb

```diff
@@ -25,6 +25,13 @@ def call(_worker, msg, _queue)
               ) do |span|
                 span.add_event('created_at', timestamp: msg['created_at'])
                 span.add_event('enqueued_at', timestamp: msg['enqueued_at'])
+
+                if msg['enqueued_at']
+                  enqueued_at = Time.at(msg['enqueued_at']).utc
+                  waiting_time = ((Time.now.utc - enqueued_at) * 1000).to_i # in ms
```

Comment:
I suppose there's no need to deal with time objects and convert them into UTC if we want to return the value in ms in the end :grin:
```suggestion
                  enqueued_at = msg['enqueued_at']
                  waiting_time = (Time.now.to_i - enqueued_at) * 1000 # in ms
```

Reply:
It won't be the same, ie. it will be only accurate down to the second, not ms, cause `enqueued_at` [is given as a float](https://github.com/mperham/sidekiq/blob/9998a59b6da6eaf8fb316f8d34c0af7eb21f4dbb/lib/sidekiq/client.rb#L103). But with `to_f` it might work, I'll try it out, thanks!